fitlins/interfaces/bids.py

```diff
@@ -167,16 +167,18 @@ def _run_interface(self, runtime):
                 block.model['HRF_variables'], mode='sparse', force=True):
             info = {}
 
-            bold_files = layout.get(type='bold',
+            space = layout.get_spaces(type='preproc')[0]
```

Comment:
I guess the argument is that any space is as good as any other? I think surface spaces (`.gii`) might end up having a weird effect. Can we add `extensions` to this line?